packages/terra-form-select/src/shared/_FrameUtil.js

```diff
@@ -139,6 +139,19 @@ class FrameUtil {
     }
     return '-1';
   }
+
+  /**
+   * Allows time for state update to render select menu DOM before shifting focus to it
+   * @param {Object} frame - The frame object.
+   */
+
+  static shiftFocusToMenu(frame) {
+    setTimeout(() => {
+      if (frame.selectMenu) {
+        frame.selectMenu.focus();
+      }
+    }, 100);
+  }
```

Comment:
Nit: As a static util method I think this should work as an agnostic utility and be passed the element to be focused or at the very least pass the item to focus as a parameter and not the frame. 

```js
  static focusMenu(menu) { // setMenuFocus, delayMenuFocus, delayFocus, setFocus
    setTimeout(() => {
      if (menu) {
        menu.focus();
      }
    }, 100);
  }
```

Reply:
I had tried passing this.selectMenu here but it was passing undefined (weird), hence did it this way.

Reply:
This is probably an issue with maintaining references. I think leaving it the way it is will be fine. My concerns were it creates implicit naming convention of instance variables between unrelated classes and passes a reference to the entire class to call focus on a dom node. This being an internal util this isn't too big of deal.